cli/fathom_web/train.py

```diff
@@ -82,18 +83,21 @@ def learn(learning_rate, iterations, x, y, validation=None, stop_early=False, ru
     return model
 
 
-def accuracy_per_tag(model, x, y):
+def accuracy_per_tag(y, y_pred):
     """Return the accuracy 0..1 of the model on a per-tag basis, given input
```

Comment:
Docstring needs an update since the params changed.

Reply:
Whoops. I will update that.